Gruntfile.js

```diff
@@ -97,17 +144,42 @@ module.exports = function (grunt) {
         }
     });
 
+    grunt.registerMultiTask('icons', function () {
+        var iconsDirContent = fs.readdirSync(grunt.template.process('icons'));
+
+        var svgFilesRegex = /\.svg$/;
+        var iconFiles = iconsDirContent
+            .filter((filename) => svgFilesRegex.test(filename))
+            .map((filename) => filename.replace(svgFilesRegex, ''));
+
+        var iconFilesObject = {};
+
+        iconFiles.forEach((icon) => {
+            iconFilesObject[icon] = icon;
+        });
+
+        grunt.file.write(grunt.template.process('<%= config.paths.jsonDir %>/icons.json'), JSON.stringify(iconFilesObject));
+    });
+
     // - - - T A S K S - - -
 
     grunt.registerTask('default', 'dev');
 
     grunt.registerTask('dev', [
+        'icons',
+        'svgstore',
+        'json-to-sass',
+        'json-to-sass-map',
         'sass',
         'nunjucks',
         'watch'
     ]);
 
     grunt.registerTask('build', [
+        'icons',
+        'svgstore',
+        'json-to-sass',
+        'json-to-sass-map',
```

Comment:
Can these 4 be ran seperately. cause u could move into a seperate task and call that here and in the dev task.